SDDC_FX3/makefile

```diff
@@ -47,25 +55,32 @@ EXES = $(MODULE).$(EXEEXT)
 $(MODULE).$(EXEEXT): $(A_OBJECT) $(C_OBJECT)
 	$(LINK)
 
+cyfxtx.c:
```

Comment:
can we avoid copy here? Mixing the code and some .c file copied in is confusing. I can easy make mistake to add them to git as well. The current makefile will compile the file at its original place.

Reply:
at least add them to .gitignore

Reply:
The README now contains instructions on how to build the firmware (if you think they are too confusing, I can split the Windows installation of the SDK and the Linux installation of the SDK into two different subsections).

As per the two files ('cyfxtx.c' and 'cyfx_gcc_startup.S') that are being copied, I ran a 'diff' between the ones in the repo and the ones in Cypress EZ-USB FX3 SDK version 1.3.4, and the second one is nearly identical (except for the Copyright year), while the first one 'cyfxtx.c' has many significant changes, so perhaps it should be updated too.